src/server/req_message.c

```diff
@@ -185,17 +186,16 @@ void
 req_py_spawn(struct batch_request *preq)
 {
 	int             jt;		/* job type */
+	int             js;     /* job state */
 	job		*pjob;
 	int		rc;
 	char		*jid = preq->rq_ind.rq_py_spawn.rq_jid;
-	int		i, offset;
 
 	/*
-	 ** Returns job pointer for singleton job or "parent" of
-	 ** an array job.
+	 ** Returns job pointer for singleton job and "parent" of
+	 ** an array sub job.
```

Comment:
I don't think we use ** inside multi-line comment now. Please use single * only.

Reply:
Not my additions. However Fixed in rc 17 https://github.com/PBSPro/pbspro/pull/590/commits/f94a3405ef13edaa87df40a89af36420b9627c7b